src/main/proto/netflix/titus/agent.proto

```diff
@@ -269,4 +269,54 @@ message ContainerInfo {
     }
     // Process to be executed inside the container
     optional Process process = 38;
+
```

Comment:
Isn't this from the master? If so, are you planning on them adding it? Alternatively, if not can we put these fields in an embedded message, saying they'll be unset by the master? Or leaving them unset is okay?

Reply:
I think using an embedded message is a good approach to indicate that it's unneeded.

Reply:
OK, done.